sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample.py

```diff
@@ -7,27 +7,28 @@
 # --------------------------------------------------------------------------
 
 """
-FILE: sms_sample.py
+FILE: send_sms_to_multiple_recipients_sample.py
 DESCRIPTION:
-    These samples demonstrate sending mutiple sms messages and resending 
-    any failed messages.
-    
-    ///authenticating a client via a connection string
+    This sample demonstrates sending an SMS message to multiple recipients. The SMS client is 
+    authenticated a client using a connection string.
```

Comment:
I think this is ungrammatical, looks like we have redundant "a client".

Reply:
Yeah, it should be ```The SMS client is authenticated using a connection string```.